yacc.go

```diff
@@ -3470,22 +3469,20 @@ func $$Parse($$lex $$Lexer) int {
 
 func ($$rcvr *$$ParserImpl) Parse($$lex $$Lexer) int {
 	var $$n int
-	var $$lval $$SymType
```

Comment:
The address is passed to the lexer so escape analysis puts it on the heap. Embedding it in `ParserImpl` ensures it is allocated with the parser and the stack and not separately.